### DISCUSSION TITLE
Terragrunt import RFC

### DISCUSSION THREAD

**Discussion by cveld** [gruntwork-io/knowledge-base#792] [Feedback]
What is the current status of the import RFC? Would be great to rationalize all the double `read_terragrunt_config()` function calls. Additionally, would be great to be able to use depedency outputs directly into local expressions.

---

<ins datetime="2023-12-11T21:34:50Z">
  <p><a href="https://support.gruntwork.io/hc/requests/110639">Tracked in ticket #110639</a></p>
</ins>

**Comment by gitsstewart** [comment]
@brikis98 or @denis256 do you have any insight into this one?

**Comment by denis256** [comment]
Hi,
it is still in design discussions

References:
https://terragrunt.gruntwork.io/docs/rfc/imports/#open-question
